src/internal/m365/collection/site/metadata.go

```diff
@@ -106,3 +106,13 @@ func pathFromPrevString(ps string) (path.Path, error) {
 
 	return p, nil
 }
+
+func makeTombstones(dps metadata.DeltaPaths) map[string]string {
```

Comment:
This along with `handleTombstones` code are some more candidates for the tech debt ticket to consolidate general funcs across services